src/components/registration/steps.jsx

```diff
@@ -91,6 +91,13 @@ module.exports = {
         },
         validateUsername: function (username, callback) {
             callback = callback || function () {};
+            if (username.length < 1) {
```

Comment:
Ah – I thought that `Input` would be handling that by sending an empty string but sounds good.